mock/service/controller.go

```diff
@@ -58,6 +63,18 @@ func (s *service) CreateVolume(
 		return nil, status.Errorf(codes.OutOfRange, "Requested capacity %d exceeds maximum allowed %d", capacity, MaxStorageCapacity)
 	}
 
+	// Simulate "out of storage space" error?
+	if capacityLimit := req.GetParameters()["capacityLimit"]; capacityLimit != "" {
```

Comment:
@jsafrane how soon is that error injection coming?

Reply:
/hold

There's now a WIP PR for that in PR #251. Using that approach makes more sense than adding special cases like this one here to the code.

Reply:
/hold cancel